Doc/c-api/unicode.rst

```diff
@@ -612,6 +612,12 @@ APIs:
    decref'ing the returned objects.
 
 
+.. c:function:: const char* PyUnicode_GetDefaultEncoding(void)
+
+   Return a ``"utf-8"`` string constant, which corresponds to the
+   :func:`~sys.getdefaultencoding` function in Python.
```

Comment:
I suggest this wording, plus making lifetime explicit since we're returning a buffer.
(Currently the string is static of course, but if it'd ever become dynamic again, it'd need to be borrowed from the interpreter.)
```suggestion
   Return the name of the default string encoding, ``"utf-8"``.
   See :func:`sys.getdefaultencoding`.

   The returned string does not need to be freed, and is valid
   until interpreter shutdown.
```